openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py

```diff
@@ -68,6 +68,19 @@ def fill_payment_id_and_statement_line_id_fields(env):
     )
 
 
+def fill_partial_reconcile_debit_and_credit_amounts(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_partial_reconcile
+        SET debit_amount_currency = amount, credit_amount_currency = amount
+        WHERE debit_amount_currency IS NULL AND credit_amount_currency is NULL
+            AND credit_currency_id = debit_currency_id
+       """,
+    )
+    # TODO: compute debit and credit amount when currencies are different
```

Comment:
Maybe you could make a request to check if the different currency caqe exists.
If yes, log an error.
Not a blocking point.